cypress/integration/dashboard_spec.js

```diff
@@ -1,4 +1,4 @@
-describe('Details', () => {
+describe('Home Page', () => {
```

Comment:
Haha, thanks for fixing this! I accidentally left Details in when I copy and pasted the describe block from a previous project.